compiler/rustc_middle/src/middle/exported_symbols.rs

```diff
@@ -35,7 +35,10 @@ pub enum SymbolExportKind {
 pub struct SymbolExportInfo {
     pub level: SymbolExportLevel,
     pub kind: SymbolExportKind,
+    /// Was the symbol marked as `#[used(compiler)]` or `#[used(linker)]`?
     pub used: bool,
+    /// Was the symbol marked as `#[rustc_std_internal_symbol]`?
+    pub rustc_std_internal_symbol: bool,
```

Comment:
The only difference between used and rustc_std_internal_symbol is whether LTO will export the symbol or not. `used` will result in the symbol being exported, while `rustc_std_internal_symbol` won't.

Reply:
> The only difference between used and rustc_std_internal_symbol is whether LTO will export the symbol or not. `used` will result in the symbol being exported, while `rustc_std_internal_symbol` won't.

Could you add this as a comment to source code.
Or perhaps turn `used: bool` into something like `used: Option<ExportedByLto>`.

Reply:
Updated the comment.